sudoku/general/onInputCheck.js

```diff
@@ -0,0 +1,20 @@
+let wrapper = document.querySelector('.board');
+let check = document.querySelector('.check');
+let hard = document.querySelector('.hard');
+let checkHard = document.querySelector('.addRandom');
+function checkNum(event) {
+  const onlyNumbers = /^[1-9]$/;
```

Comment:
This will work too. Your solution is okay, but you will miss the "0"
```suggestion
  const onlyNumbers = /^\d+$/;
```